notebook-tiny/pyspark.ipynb

```diff
@@ -435,11 +435,19 @@
     "def trimFieldImpl(js):\n",
     "    try:\n",
     "        d = json.loads(js)\n",
-    "        return [d.get(f) for f in interesting_fields]\n",
```

Comment:
Agreed, regarding the `d.get(f)` returning either a value or None. So the only realistic thing that could fail is `json.loads()`. This part of the patch can be discarded then.